### DISCUSSION TITLE
Error: Command "yarn run build" exited with 1

### DISCUSSION THREAD

**Discussion by Claudio-Lins** [vercel/community#51] [Help] [answered]
Hello guys,
First I apologize if this issue is in the wrong place.
I am new to the world of programming and also to GitHub.
I am trying to make a DEPLOY and I am not getting it because it has some errors.
Has anyone ever experienced this?

Since already thank you all very much !!

Below is the Build Log

13:33:38.374  	Cloning github.com/Claudio-Lins/linstur (Branch: dev-design, Commit: c48a190)
13:33:38.888  	Cloning completed in 514ms
13:33:38.891  	Analyzing source code...
13:33:39.293  	Installing build runtime...
13:33:41.267  	Build runtime installed: 1973.933ms
13:33:43.001  	Looking up build cache...
13:33:43.047  	Build cache not found
13:33:43.780  	Installing dependencies...
13:33:44.007  	yarn install v1.22.10
13:33:44.065  	warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
13:33:44.067  	[1/4] Resolving packages...
13:33:44.352  	[2/4] Fetching packages...
13:33:51.537  	info fsevents@2.1.3: The platform "linux" is incompatible with this module.
13:33:51.537  	info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
13:33:51.540  	info fsevents@2.3.1: The platform "linux" is incompatible with this module.
13:33:51.540  	info "fsevents@2.3.1" is an optional dependency and failed compatibility check. Excluding it from installation.
13:33:51.541  	info fsevents@1.2.13: The platform "linux" is incompatible with this module.
13:33:51.541  	info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
13:33:51.550  	[3/4] Linking dependencies...
13:33:51.555  	warning "tailwindcss > postcss-nested@5.0.3" has incorrect peer dependency "postcss@^8.1.13".
13:33:56.442  	[4/4] Building fresh packages...
13:33:57.740  	Done in 13.74s.
13:33:57.763  	Running "yarn run build"
13:33:58.032  	yarn run v1.22.10
13:33:58.056  	$ next build
13:33:58.712  	Browserslist: caniuse-lite is outdated. Please run:
13:33:58.712  	npx browserslist@latest --update-db
13:33:58.803  	info  - Creating an optimized production build...
13:33:58.818  	Attention: Next.js now collects completely anonymous telemetry regarding usage.
13:33:58.818  	This information is used to shape Next.js' roadmap and prioritize features.
13:33:58.818  	You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
13:33:58.818  	https://nextjs.org/telemetry
13:34:01.654  	Failed to compile.
13:34:01.654  	ModuleNotFoundError: Module not found: Error: Can't resolve '../components/nav' in '/vercel/workpath0/pages'
13:34:01.654  	> Build error occurred
13:34:01.655  	Error: > Build failed because of webpack errors
13:34:01.655  	    at /vercel/workpath0/node_modules/next/dist/build/index.js:15:918
13:34:01.655  	    at runMicrotasks (<anonymous>)
13:34:01.655  	    at processTicksAndRejections (internal/process/task_queues.js:97:5)
13:34:01.655  	    at async /vercel/workpath0/node_modules/next/dist/build/tracer.js:1:525
13:34:01.672  	error Command failed with exit code 1.
13:34:01.673  	info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
13:34:01.682  	Error: Command "yarn run build" exited with 1
13:34:04.455  	Done with "package.json"

**Comment by styfle** [comment, marked as answer]
Hi @Claudio-Lins 

The relevant error message is here:

> ModuleNotFoundError: Module not found: Error: Can't resolve '../components/nav' in '/vercel/workpath0/pages'

So it sounds like you are trying to import a file that doesn't exist. You'll need one of these to exist:

- `components/nav.js`
- `components/nav/index.js`

This is case sensitive so if you had a file called `components/Nav.js` you would have to import `components/Nav` to match the file.

**Reply by Claudio-Lins** [reply]
just that, thank you very much for your help!

**Reply by mahijendra** [reply]
Hey @styfle 

I've been trying to make a DEPLOY and I'm not getting it because it has some errors.
Your help would be truly appreciated .

Build logs


16:58:44 | Installing build runtime...
-- | --
16:58:49 | Build runtime installed: 4.933s
16:58:53 | Looking up build cache...
16:58:54 | Build cache not found
16:58:55 | Installing dependencies...
16:58:55 | yarn install v1.22.10
16:58:55 | [1/4] Resolving packages...
16:58:56 | [2/4] Fetching packages...
16:59:03 | info fsevents@2.3.2: The platform "linux" is incompatible with this module.
16:59:03 | info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
16:59:03 | [3/4] Linking dependencies...
16:59:03 | warning " > postcss-cli@8.3.1" has unmet peer dependency "postcss@^8.0.0".
16:59:03 | warning "postcss-cli > postcss-reporter@7.0.2" has unmet peer dependency "postcss@^8.1.0".
16:59:07 | [4/4] Building fresh packages...
16:59:07 | Done in 12.08s.
16:59:07 | Detected Next.js version: 10.1.3
16:59:07 | Running "yarn run build"
16:59:08 | yarn run v1.22.10
16:59:08 | $ next build
16:59:09 | info  - Using webpack 4. Reason: future.webpack5 option not enabled https://nextjs.org/docs/messages/webpack5
16:59:09 | info  - Checking validity of types...
16:59:09 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
16:59:09 | This information is used to shape Next.js' roadmap and prioritize features.
16:59:09 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
16:59:09 | https://nextjs.org/telemetry
16:59:09 | info  - Creating an optimized production build...
16:59:09 | > Build error occurred
16:59:09 | Error: Cannot find module 'tailwindcss'
16:59:09 | Require stack:
16:59:09 | - /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js
16:59:09 | - /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/index.js
16:59:09 | - /vercel/path0/node_modules/next/dist/build/webpack/config/index.js
16:59:09 | - /vercel/path0/node_modules/next/dist/build/webpack-config.js
16:59:09 | - /vercel/path0/node_modules/next/dist/build/index.js
16:59:09 | - /vercel/path0/node_modules/next/dist/cli/next-build.js
16:59:09 | - /vercel/path0/node_modules/next/dist/bin/next
16:59:09 | at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
16:59:09 | at Function.mod._resolveFilename (/vercel/path0/node_modules/next/dist/build/webpack/require-hook.js:4:1784)
16:59:09 | at Function.resolve (internal/modules/cjs/helpers.js:94:19)
16:59:09 | at loadPlugin (/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:1:1566)
16:59:09 | at /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:7:1801
16:59:09 | at Array.map (<anonymous>)
16:59:09 | at getPostCssPlugins (/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:7:1794)
16:59:09 | at async __overrideCssConfiguration (/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/overrideCssConfiguration.js:1:272)
16:59:09 | at async getBaseWebpackConfig (/vercel/path0/node_modules/next/dist/build/webpack-config.js:150:166)
16:59:09 | at async Promise.all (index 0)
16:59:09 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:5:584)
16:59:09 | at async /vercel/path0/node_modules/next/dist/build/index.js:11:960
16:59:09 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:5:584) {
16:59:09 | code: 'MODULE_NOT_FOUND',
16:59:09 | requireStack: [
16:59:09 | '/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js',
16:59:09 | '/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/index.js',
16:59:09 | '/vercel/path0/node_modules/next/dist/build/webpack/config/index.js',
16:59:09 | '/vercel/path0/node_modules/next/dist/build/webpack-config.js',
16:59:09 | '/vercel/path0/node_modules/next/dist/build/index.js',
16:59:09 | '/vercel/path0/node_modules/next/dist/cli/next-build.js',
16:59:09 | '/vercel/path0/node_modules/next/dist/bin/next'
16:59:09 | ]
16:59:09 | }
16:59:09 | error Command failed with exit code 1.
16:59:09 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
16:59:09 | Error: Command "yarn run build" exited with 1


:(

**Reply by styfle** [reply]
> Error: Cannot find module 'tailwindcss'

Did you forget to install tailwindcss?

**Reply by Indu-Chandana** [reply]
Thank you so much @styfle

**Reply by mustajabgohar123** [reply]
@styfle 

01:22:04.945 | Build runtime installed: 649.039ms
01:22:05.968 | Installing build runtime...
01:22:06.604 | Build runtime installed: 635.412ms
01:22:07.625 | Installing build runtime...
01:22:08.278 | Build runtime installed: 652.442ms
01:22:09.296 | Installing build runtime...
01:22:09.919 | Build runtime installed: 622.693ms
01:22:10.936 | Installing build runtime...
01:22:11.615 | Build runtime installed: 679.037ms
01:22:12.634 | Installing build runtime...
01:22:13.249 | Build runtime installed: 614.271ms
01:22:14.273 | Installing build runtime...
01:22:14.935 | Build runtime installed: 661.845ms
01:22:15.956 | Installing build runtime...
01:22:17.756 | Build runtime installed: 1.800s
01:22:23.553 | No Build Cache available
01:22:23.725 | No Build Cache available
01:22:23.733 | No Build Cache available
01:22:23.852 | No Build Cache available
01:22:23.975 | No Build Cache available
01:22:25.584 | Warning: Due to "engines": { "node": ">=10.0.0" } in your `package.json` file, the Node.js Version defined in your Project Settings ("14.x") will not apply. Learn More: http://vercel.link/node-version
01:22:25.589 | Installing dependencies...
01:22:25.666 | Warning: Due to "engines": { "node": ">=10.0.0" } in your `package.json` file, the Node.js Version defined in your Project Settings ("14.x") will not apply. Learn More: http://vercel.link/node-version
01:22:25.671 | Installing dependencies...
01:22:25.723 | Warning: Due to "engines": { "node": ">=10.0.0" } in your `package.json` file, the Node.js Version defined in your Project Settings ("14.x") will not apply. Learn More: http://vercel.link/node-version
01:22:25.729 | Installing dependencies...
01:22:25.974 | Warning: Due to "engines": { "node": ">=10.0.0" } in your `package.json` file, the Node.js Version defined in your Project Settings ("14.x") will not apply. Learn More: http://vercel.link/node-version
01:22:25.978 | Installing dependencies...
01:22:26.017 | Warning: Due to "engines": { "node": ">=10.0.0" } in your `package.json` file, the Node.js Version defined in your Project Settings ("14.x") will not apply. Learn More: http://vercel.link/node-version
01:22:26.022 | Installing dependencies...
01:22:26.066 | yarn install v1.22.11
01:22:26.181 | [1/5] Validating package.json...
01:22:26.184 | [2/5] Resolving packages...
01:22:26.682 | [3/5] Fetching packages...
01:22:45.196 | info fsevents@2.3.2: The platform "linux" is incompatible with this module.
01:22:45.198 | info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
01:22:45.204 | info fsevents@1.2.13: The platform "linux" is incompatible with this module.
01:22:45.204 | info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
01:22:45.217 | [4/5] Linking dependencies...
01:22:45.226 | warning " > parcel-plugin-bundle-manifest@0.2.0" has unmet peer dependency "parcel-bundler@^1.1.0".
01:23:04.344 | [5/5] Building fresh packages...
01:23:09.685 | info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
01:23:09.715 | Error: Command "yarn install" exited with 1
01:23:09.879 | yarn install v1.22.11
01:23:09.987 | [1/5] Validating package.json...
01:23:09.990 | [2/5] Resolving packages...
01:23:10.490 | [3/5] Fetching packages...
01:23:11.400 | info fsevents@2.3.2: The platform "linux" is incompatible with this module.
01:23:11.400 | info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
01:23:11.400 | info fsevents@1.2.13: The platform "linux" is incompatible with this module.
01:23:11.400 | info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
01:23:11.412 | [4/5] Linking dependencies...
01:23:11.427 | warning " > parcel-plugin-bundle-manifest@0.2.0" has unmet peer dependency "parcel-bundler@^1.1.0".
01:23:39.595 | [5/5] Building fresh packages...
01:23:42.765 | info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
01:23:44.464 | Error: Command "yarn install" exited with 1
01:23:44.670 | yarn install v1.22.11
01:23:44.834 | [1/5] Validating package.json...
01:23:44.837 | [2/5] Resolving packages...
01:23:45.438 | [3/5] Fetching packages...
01:23:46.357 | info fsevents@2.3.2: The platform "linux" is incompatible with this module.
01:23:46.357 | info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
01:23:46.363 | info fsevents@1.2.13: The platform "linux" is incompatible with this module.
01:23:46.363 | info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
01:23:46.378 | [4/5] Linking dependencies...
01:23:46.391 | warning " > parcel-plugin-bundle-manifest@0.2.0" has unmet peer dependency "parcel-bundler@^1.1.0".
01:24:24.114 | [5/5] Building fresh packages...
01:24:27.204 | info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
01:24:28.669 | Error: Command "yarn install" exited with 1

Suddenly my deployments giving me the  "Command "yarn install" exited with 1" error. Previously deployments were working fine.
I am so stressed up :(

**Reply by SANGNGUYEN24** [reply]
Me too

**Comment by tonniva** [comment]
i use this in vue "npm install --save vue-runtime-helpers" it ok

**Reply by ShariqKalim** [reply]
<html>
<body>


00:46:08.819 | Retrieving list of deployment files...
-- | --
00:46:10.342 | Downloading 35 deployment files...
00:46:10.967 | Analyzing source code...
00:46:12.318 | Installing build runtime...
00:46:15.016 | Build runtime installed: 2.697s
00:46:18.731 | Looking up build cache...
00:46:18.992 | Build Cache not found
00:46:20.169 | Installing dependencies...
00:46:20.414 | yarn install v1.22.17
00:46:20.479 | warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
00:46:20.482 | [1/4] Resolving packages...
00:46:20.782 | [2/4] Fetching packages...
00:46:46.972 | [3/4] Linking dependencies...
00:46:46.974 | warning "@emotion/styled > @emotion/babel-plugin@11.3.0" has unmet peer dependency "@babel/core@^7.0.0".
00:46:46.975 | warning "@emotion/styled > @emotion/babel-plugin > @babel/plugin-syntax-jsx@7.14.5" has unmet peer dependency "@babel/core@^7.0.0-0".
00:46:46.977 | warning "firebase-admin > @firebase/database-compat@0.1.1" has unmet peer dependency "@firebase/app-compat@0.x".
00:46:46.977 | warning "firebase-admin > @firebase/database-compat > @firebase/database > @firebase/auth-interop-types@0.1.6" has unmet peer dependency "@firebase/app-types@0.x".
00:46:46.980 | warning "eslint-config-next > @typescript-eslint/parser > @typescript-eslint/typescript-estree > tsutils@3.21.0" has unmet peer dependency "typescript@>=2.8.0 \|\| >= 3.2.0-dev \|\| >= 3.3.0-dev \|\| >= 3.4.0-dev \|\| >= 3.5.0-dev \|\| >= 3.6.0-dev \|\| >= 3.6.0-beta \|\| >= 3.7.0-dev \|\| >= 3.7.0-beta".
00:47:09.180 | [4/4] Building fresh packages...
00:47:09.739 | Done in 49.33s.
00:47:09.766 | Detected Next.js version: 11.1.2
00:47:09.770 | Running "yarn run build"
00:47:10.044 | yarn run v1.22.17
00:47:10.071 | $ next build
00:47:10.873 | info - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5
00:47:11.027 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
00:47:11.027 | This information is used to shape Next.js' roadmap and prioritize features.
00:47:11.027 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
00:47:11.027 | https://nextjs.org/telemetry
00:47:11.027 |  
00:47:11.075 | info - Checking validity of types...
00:47:13.943 |  
00:47:13.943 | Failed to compile.
00:47:13.944 |  
00:47:13.944 | ./pages/about.js
00:47:13.944 | 9:9 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.944 | 16:13 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.944 | 25:13 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.945 | 34:13 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.945 | 51:49 Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`. react/no-unescaped-entities
00:47:13.945 |  
00:47:13.945 | ./pages/contact.js
00:47:13.945 | 8:27 Error: React Hook "useState" is called in function "contact" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.945 |  
00:47:13.946 | ./pages/dashboard/index.js
00:47:13.946 | 12:47 Error: React Hook "useState" is called in function "index" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.946 | 24:3 Error: React Hook "useEffect" is called in function "index" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.946 | 46:11 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.946 | 53:15 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.946 | 57:13 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.947 |  
00:47:13.947 | ./pages/dashboard/tempview/one.js
00:47:13.947 | 9:18 Error: React Hook "useRouter" is called in function "one" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.947 | 10:47 Error: React Hook "useState" is called in function "one" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.947 | 29:11 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.947 | 33:9 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.947 |  
00:47:13.948 | ./pages/login.js
00:47:13.948 | 9:18 Error: React Hook "useRouter" is called in function "login" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.948 | 27:29 Error: React Hook "useState" is called in function "login" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.948 | 28:35 Error: React Hook "useState" is called in function "login" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.948 |  
00:47:13.948 | ./pages/register.js
00:47:13.948 | 9:18 Error: React Hook "useRouter" is called in function "register" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.948 | 10:29 Error: React Hook "useState" is called in function "register" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.948 | 11:35 Error: React Hook "useState" is called in function "register" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.948 | 12:31 Error: React Hook "useState" is called in function "register" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.948 | 13:41 Error: React Hook "useState" is called in function "register" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.949 | 14:27 Error: React Hook "useState" is called in function "register" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.949 | 15:37 Error: React Hook "useState" is called in function "register" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. react-hooks/rules-of-hooks
00:47:13.949 |  
00:47:13.949 | ./components/Main.js
00:47:13.949 | 8:11 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.949 | 11:51 Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`. react/no-unescaped-entities
00:47:13.949 | 13:13 Error: `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`. react/no-unescaped-entities
00:47:13.950 | 14:20 Error: `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`. react/no-unescaped-entities
00:47:13.950 | 16:11 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.950 |  
00:47:13.950 | ./components/Navbar.js
00:47:13.950 | 8:11 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.950 | 9:13 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.950 | 14:13 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.951 | 17:13 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.951 | 20:13 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.951 | 23:13 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.951 |  
00:47:13.951 | ./components/Navbar_dashboard.js
00:47:13.951 | 16:11 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.951 | 17:13 Warning: Do not use <img>. Use Image from 'next/image' instead. See https://nextjs.org/docs/messages/no-img-element. @next/next/no-img-element
00:47:13.951 | 22:13 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.951 | 27:13 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.951 | 31:13 Warning: passHref is missing. See https://nextjs.org/docs/messages/link-passhref @next/next/link-passhref
00:47:13.951 |  
00:47:13.952 | info - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
00:47:13.969 | error Command failed with exit code 1.
00:47:13.970 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
00:47:13.991 | Error: Command "yarn run build" exited with 1


</body>
</html>

**Comment by Pieparker** [comment]
`Error: Command "yarn run build" exited with 1` is an output from an erroneous build. For anybody seeing this log line, please check the log lines prior to this line in your [build logs](https://vercel.com/docs/concepts/deployments/logs#build-logs), to identify the issue with your build. 

If you continue to experience issues, consider checking out other discussions and our docs before creating your own discussion here.
